wire-ios/Wire-iOS/Sources/UserInterface/StartUI/StartUI/StartUIViewController+NavigationController.swift

```diff
@@ -0,0 +1,67 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+
+extension StartUIViewController {
+
+    func setupNavigationBar() {
+        navigationController?.navigationBar.barTintColor = backgroundColor
+        navigationController?.navigationBar.isTranslucent = false
+        navigationController?.navigationBar.tintColor = UIColor.accent()
+        navigationController?.navigationBar.titleTextAttributes = DefaultNavigationBar.titleTextAttributes()
+    }
+
+    func setupNavigationBarButtonItems() {
+        let cancelButton = UIBarButtonItem(
+            title: L10n.Localizable.General.cancel,
+            style: .plain,
+            target: self,
+            action: #selector(onDismissPressed)
+        )
+
+        cancelButton.accessibilityLabel = L10n.Accessibility.ContactsList.CanceButton.description
+        cancelButton.accessibilityIdentifier = "cancel"
+
+        navigationItem.rightBarButtonItem = cancelButton
+
+        let createGroupButton = UIBarButtonItem(
+            title: L10n.Localizable.Peoplepicker.Button.createConversation,
+            style: .plain,
+            target: self,
+            action: #selector(createGroup)
+        )
+
+        createGroupButton.accessibilityLabel = L10n.Localizable.Peoplepicker.Button.createConversation
+        createGroupButton.accessibilityIdentifier = "create_group"
+
+        navigationItem.leftBarButtonItem = createGroupButton
+    }
+
+    @objc
+    func onDismissPressed() {
```

Comment:
Can this be `private` and maybe just the name of the action like `func dismiss()`?